packages/cli/generators/relation/index.js

```diff
@@ -145,9 +149,12 @@ module.exports = class RelationGenerator extends ArtifactGenerator {
     }
 
     if (this.options[parameter]) {
+      this.isChecked[parameter] = true;
       if (!modelList.includes(this.options[parameter])) {
-        throw new Error(
-          `"${this.options[parameter]}" ${ERROR_MODEL_DOES_NOT_EXIST}`,
+        return this.exit(
+          new Error(
+            `"${this.options[parameter]}" ${ERROR_MODEL_DOES_NOT_EXIST}`,
```

Comment:
Nitpick - feel free to ignore.

This code formatting creates a long & nested structure that can be seen unnecessarily verbose. In the past, I quite liked the outcome of extracting the message into a new variable.

```ts
const msg = `"${this.options[parameter]}" ${ERROR_MODEL_DOES_NOT_EXIST}`;
return this.exit(new Error(msg));
```

This approach starts to shine in particular when the error message becomes too long for a single line.

If you decide to apply my suggestion, then please update all places in your code for consistency please.

Reply:
removed catch